src/en/charms.md

```diff
@@ -81,29 +84,34 @@ application, HAProxy, so we should also deploy a charm for that now:
 juju deploy haproxy
 ```
 
-If you check what you model currently contains by running...
+After a while if you check what your model currently contains by running...
 
 ```bash
 juju status
 ```
 
 ...you should see something like this:
 
+<!-- JUJUVERSION: 2.2.2-xenial-amd64 -->
+<!-- JUJUCOMMAND: juju status -->
 ```no-highlight
-App        Version  Status       Scale  Charm      Store       Rev  OS      Notes
-haproxy             maintenance      1  haproxy    jujucharms   37  ubuntu  
-mariadb             maintenance      1  mariadb    jujucharms    6  ubuntu  
-mediawiki           maintenance      1  mediawiki  jujucharms    5  ubuntu  
+Model  Controller         Cloud/Region     Version  SLA
+prod   google-controller  google/us-east1  2.2.2    unsupported
```

Comment:
how did the model come to be called 'prod'? I thought this runthrough used the automatically created 'default' model?

Reply:
I can change it to 'default' if you want. Much wasn't working over here and I was doing a lot of destroying and adding of models. Sometimes I would have multiple models. The 'prod' is what I had when it worked for me, that's all.

Reply:
I think it makes more sense as 'default'